app/lib/max_allowed_days_loader.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module MaxAllowedDaysLoader
+  module_function
+
+  def load_account_suspension
+    config[:account_suspension]
+  end
+
+  def load_account_inactivity
+    config[:account_inactivity]
+  end
+
+  def load_contract_unpaid_time
+    config[:contract_unpaid_time]
+  end
+
+  def config
+    @config ||= load_config
+  end
+
+  def valid_configuration?
+    config && @valid_configuration
+  end
+
+  def load_config
+    config = (ThreeScale.config.max_allowed_days || {}).each_with_object({}) do |(key, value), collection|
+      collection[key.to_sym] = value.public_send(:days) if value.to_i >= 1
+    end
+    @valid_configuration = %i[account_suspension account_inactivity contract_unpaid_time].all? { |key| config.key?(key) }
```

Comment:
Checking if the `config.keys` contains %i[account_suspension account_inactivity contract_unpaid_time] and that all the values are integer is enough I think

Reply:
I agree except that imo they need to be `>= 1`. First of all negative values for sure it is a bad idea because it is the future. But even near 0 means it can be scheduled for deletion the same day... 😕 Even 1 hour later... I'm not sure if it is asking for problems... 😕 
WDYT @hallelujah ?

Reply:
It is the responsibility of the config. If it is in the future then it is equivalent to never suspended and never deleted right?
That sounds OK for me.
If it is 0 then OK fro

Reply:
Ok, makes sense 😄 👍

Reply:
I don't really see a problem with negative values. Yes, it will be in the future and that date will be compared agains events always in the past (`state_changed_at`, `created_at`). This is virtually equivalent to no time gap at all, right?

Reply:
It means never happening for accounts (we won't have accounts that their state has been changed in the future 😂 ), and no time gap at all for the contract, because it means `that it has not paid in the future yet` so all of them 😄